docs/content/latest/releases/whats-new/latest-release.md

```diff
@@ -24,6 +24,304 @@ Refer to [issue 8531](https://github.com/yugabyte/yugabyte-db/issues/8531) for d
+* [[9128](https://github.com/yugabyte/yugabyte-db/issues/9128)] [DocDB] Converted MetaCache::DoLookupAllTablets excessive INFO logs to VLOGs
+* [[9146](https://github.com/yugabyte/yugabyte-db/issues/9146)] [YSQL] add "/HASH" to "ASC/DESC" errmsg
+* [[9154](https://github.com/yugabyte/yugabyte-db/issues/9154)] [[2519](https://github.com/yugabyte/yugabyte-db/issues/2519)] [[1973](https://github.com/yugabyte/yugabyte-db/issues/1973)] Initial changes for supporting the aarch64 architecture
+* [[9185](https://github.com/yugabyte/yugabyte-db/issues/9185)] Use atomic write for created table and its tablets
+* [[9208](https://github.com/yugabyte/yugabyte-db/issues/9208)] [YSQL] Change function call information to be fixed length
+* [[9265](https://github.com/yugabyte/yugabyte-db/issues/9265)] [YBase] Return error if tablet is not removed by RemoveRunningTablet
+* [adhoc] [DocDB] Remove unused argument from Tablet::NewRowIterator
+* [DocDB] Add Clang thread safety annotations to catalog_entity_info (#8186)
+* [SMARTDRIVER] [YSQL] Function to fetch database servers #7879
+* [YBase] Use SharedLock in GetNumRelevantReplicas
+* [YSQL] Import Fix incorrect return value in pg_size_pretty(bigint)
+* [YSQL] Import Reduce the cost of planning deeply-nested views.
+* [YSQL] Import Relax transactional restrictions on ALTER TYPE ... ADD VALUE (redux).
+* [YSQL] improve ALTER TABLE [NOT] OF errmsg
+* [YSQL] improve ALTER TABLE [RE]SET errmsg
+
+#### Point-in-time recovery
+
+* [[8417](https://github.com/yugabyte/yugabyte-db/issues/8417)] PITR: Implement delete_snapshot_schedule
+* [[8418](https://github.com/yugabyte/yugabyte-db/issues/8418)] Add filter to list_snapshot_schedules
+* [[8419](https://github.com/yugabyte/yugabyte-db/issues/8419)] Consistent restore failover
+* [[8419](https://github.com/yugabyte/yugabyte-db/issues/8419)] Consistent restore for non transactional writes
+* [[8419](https://github.com/yugabyte/yugabyte-db/issues/8419)] Consistent restore for transactional writes
+* [[8543](https://github.com/yugabyte/yugabyte-db/issues/8543)] PITR: Add test for need to increase table version on restore
+* [[8773](https://github.com/yugabyte/yugabyte-db/issues/8773)] PITR: Add DDL log
+* [[8855](https://github.com/yugabyte/yugabyte-db/issues/8855)] Logging improvements for backup restore issue.
+* [[8892](https://github.com/yugabyte/yugabyte-db/issues/8892)] backups: Add timing summary to end of yb_backup
+* [[8894](https://github.com/yugabyte/yugabyte-db/issues/8894)] backups: Allow disabling of checksums in yb_backup
+* [[9046](https://github.com/yugabyte/yugabyte-db/issues/9046)] PITR: Fix crash when using multiple masters
+* [[9157](https://github.com/yugabyte/yugabyte-db/issues/9157)] [DocDB] PITR: Fix operations filters data race
+* [[9171](https://github.com/yugabyte/yugabyte-db/issues/9171)] PITR: Fix restoring snapshot to time before history cutoff
+* [backup] Added a check for 'num_tablets' in CatalogManager::ImportSnapshot.
```

Comment:
I think this is less PITR and just generic backup related, so could move to normal db improvements